index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<link rel="stylesheet" type="text/css" href="css/quiz.css">
+	<title>My Way</title>
+</head>
+<body>
+<!-- <body img src ="images/background-img.jpg"> -->
+  <Header>
+  	  <h1>Julia Kim-Chung: My Portfolio</h1>
+  </Header>
+   <div><img class = "firstPic" alt="background-pic" src ="images/background-img.jpg"></div>
+  	 <article>
+  		<fieldset class = "f1">
+  		  
+  			<legend >My Story</legend>
+  			<img  src ="images/IMG_2364.jpg"  alt="myPic" width="200" height ="250">
+  			  <p class="work-history">Work History</p>
+  			  	<ul>
+  			  	
+
+  			  		<li>
+  			  			Hotel Management/HR at the Wilshire Grand Hotel & Centre, Los Angeles, CA
+
+  			  		</li>
+
+  			  		<li>
+  			  			Retail Operation Manger/ HR Manger at juice Nashville, Nashville, TN
+  			  		</li>
+
+
+  			  	
+  			  	</ul>
+
+  			  	<p class="my-interest">My Interest</p>
+  			  		<ul>
+  			  			
+  			  			<li>	<a href = "https://www.google.com/search?q=world%27s+best+place+to+swim+photo&espv=2&biw=759&bih=805&tbm=isch&tbo=u&source=univ&sa=X&ved=0ahUKEwictoyYlOXPAhUP4GMKHWMQAjgQsAQIGw">Swimming,</a><a href = "https://www.google.com/search?q=world%27s+best+place+to+hike+photo&espv=2&biw=1680&bih=909&tbm=isch&imgil=HHZmjMFrKaCTMM%253A%253BxiZYghJ-EVacjM%253Bhttp%25253A%25252F%25252Fadventure.nationalgeographic.com%25252Fadventure%25252Ftrips%25252Fbest-trails%25252Fworlds-best-hikes-dream-trails%25252F&source=iu&pf=m&fir=HHZmjMFrKaCTMM%253A%252CxiZYghJ-EVacjM%252C_&usg=__bffV_hFTk5oJVyKnmsX18L492mo%3D&ved=0ahUKEwixwv-zkeXPAhVC0mMKHchxAfsQyjcITA&ei=a34GWPHgDsKkjwPI44XYDw#imgrc=HHZmjMFrKaCTMM%3A"> Hiking</a> with my husband and our daughter.
+  			  			</li>
+  			  		</ul>
+
+
+  		</fieldset>
+
+  		
+
+	</article>
+
+  	<article>
+
+  	  <fieldset class = "f2">
+  	    <legend> My Projects</legend>
+    	   <select name="project">
+  		
+			 
+  					<option>Project1</option>
+  					<option>Project2</option>
+  				
+  				
+   			 
+
+   		   </select>	
+
+	       
+       </fieldset>
+           
+    </article>
+
+    <article>
+
+      <fieldset class="f3">
+      		<legend>Connect With Me Via</legend>
+
+      			<p>Email: julhkm@gmail.com</p>
+
+      			<p><a href ="http://juliakimchung.github.io">Github </a><p>
+      			<p>phone: 615-999-2479</p>
+      	
+      </fieldset>
+    </article> 
+    <footer></footer>
```

Comment:
If you end up with an unused tag like this, be sure to remove it before doing a PR